src/RestControllers/AuthorizationController.php

```diff
@@ -489,14 +490,14 @@ public function oauthAuthorizationFlow(): void
 
     public function getAuthorizationServer(): AuthorizationServer
     {
-        $customClaim = new ClaimSetEntity('', ['']);
+        $customClaim = [new ClaimSetEntity('fhirUser', ['fhirUser'])];
         if (!empty($_SESSION['nonce'])) {
             // nonce scope added later. this is for id token nonce claim.
-            $customClaim = new ClaimSetEntity('nonce', ['nonce']);
+            $customClaim[] = new ClaimSetEntity('nonce', ['nonce']);
         }
 
         // OpenID Connect Response Type
-        $responseType = new IdTokenResponse(new IdentityRepository(), new ClaimExtractor([$customClaim]));
+        $responseType = new IdTokenResponse(new IdentityRepository(), new ClaimExtractor($customClaim));
```

Comment:
There are fou default claims to be allowed: 'profile', 'email', 'address', 'phone'
So if a scope doesn't have a claim here then, will not populate in the id_token.
@adunsulag this is what i've been talking about. scope to claim validation for inclusion.

Reply:
@sjpadgett  Yea, I saw that when I dug into the source code for IdTokenResponse.